src/main/scala/nl/knaw/dans/easy/fedoratobag/EasyFedoraToBagApp.scala

```diff
@@ -163,9 +163,9 @@ class EasyFedoraToBagApp(configuration: Configuration) extends DebugEnhancedLogg
       _ <- (bagDir1 / "metadata").list.toList
         .filter(_.name.toLowerCase.contains("license"))
         .traverse(file => copy(file.name, bag2))
-      fileItems <- datasetInfo.nextFileInfos.toList.traverse(addPayloadFileTo(bag2))
-      _ <- checkNotImplemented(fileItems, logger)
-      _ <- addXmlMetadataTo(bag2, "files.xml")(filesXml(fileItems))
+      fileItems <- datasetInfo.nextFileInfos.toList.traverse(addPayloadFileTo(bag2, isOriginalVersioned = true))
```

Comment:
`bag2` is a parameter for several method calls.  You could define it as an `implicit` variable outside for comprehension.